modules/packages/Sort.chpl

```diff
@@ -546,897 +546,1022 @@ iter sorted(x, comparator:?rec=defaultComparator) {
     yield i;
 }
 
-
-/* Sort Functions */
-
 /*
    Sort the 1D array `Data` in-place using a sequential bubble sort algorithm.
 
+   .. note:: This method is deprecated.
```

Comment:
Technically these are procedures (or more broadly functions) right?

Reply:
Would it be appropriate to redirect a user to the top-level `sort` function now? e.g.

```
.. note:: This procedure is deprecated - please use sort
```